src/main/java/cloud/fogbow/rcs/constants/ConfigurationPropertyDefaults.java

```diff
@@ -3,4 +3,5 @@
 public class ConfigurationPropertyDefaults {
     // RCS CONF DEFAULTS
     public static final String BUILD_NUMBER = "[testing mode]";
+    public static final String CACHE_EXPIRATION_TIME_DEFAULT = "3600";
```

Comment:
Use **TimeUtil**,  for example, Long.toString(TimeUnit.MINUTES.toMinutes(3)); // the reference value is 3 seconds...

Reply:
And separate this constant with a comment, for example, // CACHE CONF DEFAULT.

Reply:
Where can I find this TimeUtil?

Reply:
In the package java.util.concurrent.TimeUnit.